www/includes/easyparliament/templates/html/alert/index.php

```diff
@@ -0,0 +1,371 @@
+      ?>
+        <div class="alert-section alert-section--disambiguation">
+            <div class="alert-section__primary">
+
+              <?php if (isset($members) && $members->rows() > 0) { ?>
+                <h3>Sign up for alerts when people matching <i><?= _htmlspecialchars($alertsearch) ?></i> speak</h3>
+                <ul>
+                  <?php
+                    $q = $members;
+                    for ($n=0; $n<$q->rows(); $n++) {
+                  ?>
+                    <li>
+                        <form action="<?= $actionurl ?>" method="post">
+                            <input type="hidden" name="t" value="<?= _htmlspecialchars($token) ?>">
+                            <input type="hidden" name="email" value="<?= _htmlspecialchars($email) ?>">
+                            <input type="hidden" name="pid" value="<?= $q->field($n, 'person_id') ?>">
+                            When
+                            <?= member_full_name($q->field($n, 'house'), $q->field($n, 'title'), $q->field($n, 'given_name'), $q->field($n, 'family_name'), $q->field($n, 'lordofname') ) ?>
+                            <?php if ($q->field($n, 'constituency')) { ?>
+                                ( <?= $q->field($n, 'constituency') ?> )
+                            <?php } ?>
+                            speaks.
+                            <input type="submit" class="button small" value="Subscribe"></form>
+                        </form>
+                    </li>
+                  <?php } ?>
+                </ul>
+              <?php } ?>
+
+              <?php if (isset($constituencies) && count($constituencies) > 0) { ?>
+                <h3>Sign up for alerts when MPs for constituencies matching <i><?= _htmlspecialchars($alertsearch) ?></i> speak</h3>
+                <ul>
+                <?php foreach ($constituencies as $constituency => $member) { ?>
+                    <li>
+                        <form action="<?= $actionurl ?>" method="post">
+                            <input type="hidden" name="t" value="<?= _htmlspecialchars($token) ?>">
+                            <input type="hidden" name="email" value="<?= _htmlspecialchars($email) ?>">
+                            <input type="hidden" name="pid" value="<?= $member->person_id() ?>">
+                            <?php if ($valid_postcode) { ?>
+                                <input type="hidden" name="pc" value="<?= _htmlspecialchars($alertsearch) ?>">
+                            <?php } ?>
+                            <?= $member->full_name() ?>
+                            ( <?= _htmlspecialchars($constituency) ?> )
+                            <input type="submit" class="button small" value="Subscribe"></form>
+                        </li>
+                <?php } ?>
+                </ul>
+              <?php } ?>
+
+              <?php if ($alertsearch) { ?>
+                <h3>Great! Can you just confirm what you mean?</h3>
```

Comment:
If there have been name matches, but the person wants to sign up for mentions of that name, this heading is confusing. You get a list of name matches, then this heading.